src/development/LocalTest/Models/StartAppModel.cs

```diff
@@ -43,6 +44,16 @@ public class StartAppModel
         /// </summary>
         public bool InvalidTestDataPath { get; set; }
 
+        /// <summary>
+        /// 
+        /// </summary>
+        public string LocalAppUrl { get; set; }
+
+        /// <summary>
+        ///
```

Comment:
empty summaries. These should be populated with short descriptions of the properties.

Reply:
I just tried to keep the style of the file.

Reply:
Added (non)descriptive summaries for all properties in the file.

Reply:
I would like to ask you to give it another try, and with descriptive summaries this time.
Some tips along the way: 
- Developers referencing the property do not nessecarily need to know where the value is populated from, rather what it represents
- Unless, it isn't reccomended to use the property in a specific context, the model comment does not need to instruct where the property will be utilized. E.g. for the list of test apps, `A list of available test applications` would be a sufficent comment, no need to add that it will populate a dropdown, as this might change in the future.